specs/specCRUDBooster.ts

```diff
@@ -1,109 +1,132 @@
-    browser.wait(protractor.ExpectedConditions.alertIsPresent(), 1000);
-    const alert = browser.switchTo().alert();
-    expect(alert.getText()).toEqual('The stock must not be null or empty');
-    alert.accept();
-  });
-
-  it('Set empty stock and name', () => {
-    const crudPage = new CrudPage();
-    crudPage.get();
-    crudPage.clearFruitName();
-    crudPage.clearFruitStock();
     crudPage.setFruitName('');
-    crudPage.setFruitStock('');
     crudPage.clickSaveChanges();
-    browser.wait(protractor.ExpectedConditions.alertIsPresent(), 1000);
+    const EC = protractor.ExpectedConditions;
+    browser.wait(EC.alertIsPresent(), 1000);
     const alert = browser.switchTo().alert();
-    expect(alert.getText()).toEqual('The name must not be null or empty');
+    expect(alert.getText()).toEqual('The name is required!');
     alert.accept();
   });
 
-  it('Set some string to stock', async () => {
-    const crudPage = new CrudPage();
-    crudPage.get();
-    const prewStockValue = await crudPage.getFruitStockValue();
-    crudPage.setFruitStock('abcd');
-    const EC = protractor.ExpectedConditions;
-    expect(EC.textToBePresentInElementValue(
-        crudPage.getFruitStockElement(), prewStockValue.toString()));
-  });
-
   it('Create edit and delete fruit with name', () => {
     const crudPage = new CrudPage();
     crudPage.get();
     const name = 'Peach';
-    const stock = '100';
     crudPage.clearFruitName();
-    crudPage.clearFruitStock();
     crudPage.setFruitName(name);
-    crudPage.setFruitStock(stock);
     crudPage.clickSaveChanges();
     const EC = protractor.ExpectedConditions;
-    browser.wait(
-        EC.visibilityOf(crudPage.getElementByNameAndStock(name, stock)));
-    crudPage.get();
-
-    const newStock = '10';
-    crudPage.clickEditOnFruit(name);
-    crudPage.clearFruitStock();
-    crudPage.setFruitStock(newStock);
-    crudPage.clickSaveChanges();
-    browser.wait(
-        EC.visibilityOf(crudPage.getElementByNameAndStock(name, newStock)));
+    browser.wait(EC.visibilityOf(crudPage.getElementByName(name)));
     crudPage.get();
 
     const newName = 'Blackberry';
     crudPage.clickEditOnFruit(name);
     crudPage.clearFruitName();
     crudPage.setFruitName(newName);
     crudPage.clickSaveChanges();
-    browser.wait(
-        EC.visibilityOf(crudPage.getElementByNameAndStock(newName, newStock)));
+    browser.wait(EC.visibilityOf(crudPage.getElementByName(newName)));
     crudPage.get();
 
     crudPage.clickEditOnFruit(newName);
     crudPage.clearFruitName();
-    crudPage.clearFruitStock();
     crudPage.setFruitName(name);
-    crudPage.setFruitStock(stock);
     crudPage.clickSaveChanges();
-    browser.wait(
-        EC.visibilityOf(crudPage.getElementByNameAndStock(name, stock)));
+    browser.wait(EC.visibilityOf(crudPage.getElementByName(name)));
     crudPage.get();
 
     crudPage.clickRemoveOnFruit(name);
   });
+
+  it('Create edit and delete fruit with name and stock',
```

Comment:
Can you just check if `isStockPresentInRuntime` once before this line, and if `false`, then `return`?

Reply:
It is possible, but I will have to do that before any test which works with stock, because if I don't not return null value then I will return a function which contains specific test case

Reply:
Plus I didn't find any better way how programmatically skip a test